napari/layers/_labels_layer/model.py

```diff
@@ -507,6 +501,21 @@ def get_message(self, coord, value):
 
         return msg
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current image data and colors.
+        """
+        zoom_factor = np.divide(self._thumbnail_shape[:2],
+                                self._image_view.shape[:2]).min()
+        thumbnail = np.round(ndi.zoom(self._image_view, zoom_factor))
```

Comment:
see my comments above for alternative parameters to `ndi.zoom` that I think we should use in both layers but definitely we should use in this one!

Reply:
fixed, definitely an improvement, especially here